draft-secure-credential-transfer.md

```diff
@@ -211,6 +213,30 @@ Sender device may terminate the secure credential transfer by deleting the mailb
 ~~~
 {: #stateful-flow-image title="Sample stateful workflow"}
 
+## Credential Vertical in share url
+
+When a user interacts with a share URL on a Receiver device it can be helpful to know what Credential Vertical this share is for. This is particularly important if the Receiver device has multiple applications that can handle a share URL. For example, a Receiver device might want to handle a general access share in their wallet app, but handle car key shares in a specific car application.
+
+To properly route a share URL, the sender can include the Credential Vertical in the share URL as a query parameter. The Credential Vertical can't be included in the encrypted payload because the Receiver device might need to open the right application before retrieving the secure payload. The Credential Vertical query parameter uses the "v" key and supports the below types. If no Credential Vertical is provided it will be assumed that this is a general access share URL.
+
+| Vertical       | Value |
+| --------       | ----- |
+| General Access | a or *None*
+| Home Key       | h
+| Car Key        | c
+
+Share url structure:
+```
+http://relayserver.com/v1/m/{mailboxIdentifier}?v={vertical}
+```
+
+Example car key share url:
+```
+http://relayserver.com/v1/m/2bba630e-519b-11ec-bf63-0242ac130002?v=c
+```
+
+The Credential Vertical query parameter should be added to the share URL by the Sender device when constructing the full share URL that is going to be sent to the Receiver device.
```

Comment:
This sentence sounds like the credential Vertical query parameter is mandatory but the previous section mentions that "If no Credential Vertical is provided it will be assumed that this is a general access share URL."

Should we just make it mandatory consistently across the sections? I don't see why the sender device do not why to specify this

Reply:
Good point. I reworded "should" to "can", and then I think that with the table above should make it clear that it is optional.